python/exploits/VulnerableComponents.py

```diff
@@ -67,6 +75,29 @@ def create_folder(wd_path, folder_name):
     return cwd_folder
 
 
+def vulnerable_components_fix_passfail(juiceshop_url):
+    """Checks if the Vulnerable Components security risk has been successfully
+    patched for Juice Shop. Returns True if legal.md contents remain the same,
+    or False if the contents have been changed."""
+    new_zip_text = replicate_vulnerable_components(juiceshop_url)
+    legal_url = juiceshop_url+r"/ftp/legal.md"
+    try:
+        response = requests.get(legal_url)
+        response_text = response.text
+    except requests.exceptions.Timeout as et:
+        return "Timeout Error"
+    except requests.exceptions.HTTPError as eh:
+        return "HTTP Error"
+    except requests.exceptions.ConnectionError as ec:
+        return "Connection Error"
+
+    fix_check = True
+    if new_zip_text in response_text:
+        fix_check = False
+
+    return fix_check
+
+
 if __name__ == '__main__':
-    url = r'http://localhost:3000'
-    print(replicate_vulnerable_components(url))
+    url = r'http://localhost:3001'
```

Comment:
If the port is set to 3001, won't the exploit fail outright?

Reply:
Changed the port to 3000 now. 

I was testing a local vulnerable Juice Shop on port 3001 and was testing it last.